cgroup2/manager.go

```diff
@@ -504,6 +504,32 @@ func (c *Manager) Procs(recursive bool) ([]uint64, error) {
 	return processes, err
 }
 
+func (c *Manager) Threads(recursive bool) ([]uint64, error) {
+	var threads []uint64
+	err := filepath.Walk(c.path, func(p string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if !recursive && info.IsDir() {
+			if p == c.path {
+				return nil
+			}
+			return filepath.SkipDir
+		}
+		_, name := filepath.Split(p)
+		if name != cgroupThreads {
+			return nil
+		}
+		procs, err := parseCgroupProcsFile(p)
+		if err != nil {
+			return err
+		}
+		threads = append(threads, procs...)
+		return nil
+	})
```

Comment:
Given this is shared between `Procs` and this, can we split this into a helper that just takes in the name to care about? e.g. pass in cgroupThreads for this and cgroupProcs for `Procs`

Reply:
Hello, if you mean something like [this](https://github.com/Rouzip/runc/blob/50e7b9cb9605aa3409f96267aa341b15f24f862d/libcontainer/cgroups/utils.go#L140-L162)?